### ISSUE TITLE
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe4' in position 10: ordinal not in range(128)

### ISSUE THREAD

**Issue by codingjoe** [viewflow/viewflow#106]
```
Stacktrace (most recent call last):

  File "django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "newrelic/hooks/framework_django.py", line 499, in wrapper
    return wrapped(*args, **kwargs)
  File "django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "viewflow/flow/task_view.py", line 41, in __call__
    return self.func(request, **kwargs)
  File "viewflow/views/task.py", line 225, in dispatch
    return super(AssignView, self).dispatch(request, *args, **kwargs)
  File "newrelic/hooks/framework_django.py", line 872, in wrapper
    return wrapped(*args, **kwargs)
  File "django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "viewflow/views/task.py", line 212, in post
    'assigned to {}'.format(request.user.get_full_name() or request.user.username))
```

**Comment by kmmbvnr** [comment]
Python 2.7 only issue.

Fixed in `pro 0.9.2`

**Comment by codingjoe** [comment]
@kmmbvnr thanks
